uaa-server/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM adoptopenjdk/openjdk11 as source
+RUN apt-get update && apt-get install -y git
+RUN git clone -b v75.0.0 https://github.com/cloudfoundry/uaa.git
+COPY uaa.yml /uaa/uaa/src/main/resources/uaa.yml
+WORKDIR /uaa
+RUN /uaa/gradlew
+
+FROM source as run
```

Comment:
I don't think you're getting any benefit from a multi-step build here.  Unless I'm missing something, you could probably simplify using a single FROM.